frontend/src/app/main/data/workspace/groups.cljs

```diff
@@ -101,7 +101,7 @@
 
         ;; Shapes that are in a component, but are not root, must be detached,
         ;; because they will be now children of a non instance group.
-        shapes-to-detach (filter ctk/in-component-copy-not-root? shapes)
+        shapes-to-detach (filter ctk/in-component-copy-not-head? shapes)
```

Comment:
I think shapes inside a copy should not be allowed to be grouped?

Reply:
Oh, true. And ungrouped, masked, and transformed to boolean! Good catch